api/workAllocation/interfaces/task.ts

```diff
@@ -60,10 +60,25 @@ export interface Location {
     services: string[]
 }
 
+export interface LocationApi {
+    location_id: string
+    location: string
+    is_primary: boolean
+    services: string[]
+}
+
 export interface Caseworker {
```

Comment:
What is this one for now that there's a `CaseworkerApi` interface that looks like it replaces it?

Reply:
So the CaseworkerApi is the type that refers to the caseworker object given via the API, Caseworker is the class that gets the properties via the CaseworkerApi in order to return a Caseworker class object. So non-specified API information -> CaseworkerApi -> Caseworker (same with LocationApi and Location).

Reply:
Is there any way we can just have one interface? Is it simply because of *camel vs snake*? If so, we could make the appropriate changes wherever the interfaces are used within Angular and do away with the need for translation from one to another.

That could go in a separate tech debt ticket but definitely worth looking at, I think.

Reply:
CaseworkerApi is only used for defining details received via API and is then immediately used to define a Caseworker object (same with LocationApi -> Location). Happy to make a ticket for it to investigate and potentially edit.